front/pages/api/w/[wId]/assistant/conversations/[cId]/messages/[mId]/upload_raw_content_fragment.ts

```diff
@@ -0,0 +1,186 @@
+import type { WithAPIErrorReponse } from "@dust-tt/types";
+import { isContentFragmentType } from "@dust-tt/types";
+import { Storage } from "@google-cloud/storage";
+import { IncomingForm } from "formidable";
+import fs from "fs";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { getConversation } from "@app/lib/api/assistant/conversation";
+import { Authenticator, getSession } from "@app/lib/auth";
+import { ContentFragmentResource } from "@app/lib/resources/content_fragment_resource";
+import { apiError, withLogging } from "@app/logger/withlogging";
+
+const { DUST_PRIVATE_UPLOADS_BUCKET = "dust-test-data", SERVICE_ACCOUNT } =
```

Comment:
😱 feel free to use the the new config system and add it in this file:

https://github.com/dust-tt/dust/blob/main/connectors/src/resources/storage/config.ts